test/parallel/parallel.status

```diff
@@ -19,3 +19,11 @@ test-tick-processor                  : PASS,FLAKY
 [$system==solaris] # Also applies to SmartOS
 
 [$system==freebsd]
+
+# fs-watch currently needs special configuration on AIX and we
+# want to improve under https://github.com/nodejs/node/issues/5085.
+# Tests are disabled so CI can be green and we can spot other
+# regressions until this work is complete
+[$system==aix]
+test-fs-watch-enoent                 : FAIL, PASS
+test-async-wrap-check-providers      : FAIL, PASS
```

Comment:
It might be better with test-async-wrap-check-providers to handle it in the test itself, otherwise you might be missing out on some real regressions (it's a test that tests a number of subsystems, `fs.watch()` is just one of them.)

If you do, you should note it in the comment here so it's not forgotten about.